migration-engine/migration-engine-tests/tests/schema_push/mod.rs

```diff
@@ -0,0 +1,110 @@
+use migration_engine_tests::sql::*;
+use sql_schema_describer::ColumnTypeFamily;
+
+const SCHEMA: &str = r#"
+model Cat {
+    id Int @id
+    boxId Int?
+    box Box? @relation(fields: [boxId], references: [id])
+}
+
+model Box {
+    id Int @id
+    material String
+}
+"#;
+
+#[test_each_connector]
+async fn schema_push_happy_path(api: &TestApi) -> TestResult {
+    api.schema_push(SCHEMA)
```

Comment:
Schema push is all about iteration as far as i understand. Therefore we should have a happy path test case that involves 2 schema pushes imo. So i would either extend this test case or add a separate one. Just to be sure that this works as well. (And it will)